BUILD

```diff
@@ -132,6 +152,12 @@ pkg_tar(
     tags = ["container"],
 )
 
+pkg_tar(
+    name = "layer_tini_arm64v8",
+    srcs = [":tini.binary-arm64v8"],
+    tags = ["container"],
+)
+
```

Comment:
Could we have absorbed this into `pkg_tar(name="layer_tini")` and use a `select()` to pick the `srcs`?

example:
```starlark
pkg_tar(
    name = "layer_tini",
    srcs = select({
        "@platforms//cpu:arm": [":tini.binary-arm64v8"],
        "//conditions:default": [":tini.binary"],
    }),
    tags = ["container"],
)
```